rfcs/text/0041-asset-integration.md

```diff
@@ -1,7 +1,7 @@
 # 0041: Asset Integration
 <!-- Leave this ID at 0000. The ECS team will assign a unique, contiguous RFC number upon merging the initial stage of this RFC. -->
 
-- Stage: **0 (strawperson)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
+- Stage: **1 (Draft)** <!-- Update to reflect target stage. See https://elastic.github.io/ecs/stages.html -->
```

Comment:
Are we still targeting stage 1 here @SourinPaul? You mentioned stage 2 in a different conversation.

Reply:
@ebeahan, thanks for the ping. We are currently in stage 1 and targeting stage 2. 

Do I need to update this section? Please advise, or feel free to update before merging.